xml/XSLT/README

```diff
@@ -52,15 +52,19 @@ decoder.xsl     used as a stylesheet by decoder XML files for web display;
 DecoderID.xsl   used by Ant "id" target and used as an XML stylesheet to 
                 display the decoderIndex.xml file.
                 
-panelfile.xsl   stylesheet for JMRI panel files
+panelfile[VERSION].xsl   stylesheet for JMRI panel files
 
 locomotive.xsl  stylesheet for JMRI roster entry files
 
 roster.xsl      stylesheet for JMRI roster index
 
 rpsfile.xsl     stylesheet for JMRI RPS files
 
-throttle.xsl    stylesheet for JMRI throttle layout files
+throttle-config.xsl    stylesheet for individual JMRI throttle layout files
+                [Uses throttle-include.xsl]
+
+throttle-layout-config.xsl    stylesheet for JMRI_ThrottlePreference.xml files
```

Comment:
@dsand47
 Can you please send me an example of a "user named throttle layout" that uses throttle-layout-config.xsl rather than throttle-config.xsl so I can test my xsl against it?  Your example files in #10831 all reference throttle-config.xsl in line 2.  Thanks.

Reply:
A user named "throttle layout" is identical to the default "throttle layout".  The only difference is that when selecting "Save Throttles Layout..." a file name has to be supplied.

Reply:
I think throttle-config.xsl phrase something like "individual throttle config files".

throttle-layout-config.xsl is "default (JMRI_ThrottlePreference.xml) and user-named layout config files".

Reply:
Done.